src/SearchResults.js

```diff
@@ -0,0 +1,52 @@
+import FakeBookings from "./data/fakeBookings.json";
+import moment from "moment";
+
+const SearchResults = props => (
+  <table className="table table-striped">
+    <CreateTableHeader />
+    <CreateEachRow results={FakeBookings} />
+  </table>
+);
+
+const CreateTableHeader = () => (
```

Comment:
Really really nice idea here, I like it. Small adjustment would be to name this `TableHeader` as it literally _is_ the tableheader rather than `CreateTableHeader`. Another small adjustment would be to create a `Table` component which uses the `TableHeader` and `TableRow` compeonnts. e.g.

```
const SearchResults = () => {
   return <Table />
}

const Table = () => {
   return (
      <table className=''>
         <TableHeader />
         <TableRow data={fakeBookings}
      </table>
   )
}
```